### DISCUSSION TITLE
[Question]: Confirmed working cameras with PTZ Autotracking

### DISCUSSION THREAD

**Discussion by grainsoflight** [blakeblackshear/frigate#9903] [Ask A Question]
### What is your question:

What cameras has everyone been using that are confirmed to work with autotracking, or confirmed to include the "FOV Relative Move" parameter? I've been looking at several though I've been getting responses from their sales teams that they don't include the correct ONVIF parameters.

**Comment by hawkeye217** [comment]
There's a [growing list in the docs](https://docs.frigate.video/configuration/cameras/#onvif-ptz-camera-recommendations), but generally mid-range Dahua and higher end Amcrest cameras have been reported as supporting the needed ONVIF parameters.
